### PR TITLE
Bookstore: add reducers and actions

### PR DESCRIPTION
**Pull request by lilskyex0x**
### Bookstore 
🙇‍♂️Kindly give your code review to this repository.

In this PR, I made these changes:

- [ ] Added Redux Toolkit (npm install react-redux @reduxjs/toolkit).
- [ ] Created a directory that will contain all Redux logic (/src/redux)
- [ ] Configured a Redux store (/src/redux/store.js)
- [ ] Defined a slice of state for books that:
   - Includes an array of books (initial state: empty array)
   - Includes a reducer that adds a book
   - Includes a reducer that removes a book
- [ ] Defined a slice of state for categories that:
   - Includes an array of categories (initial state: empty array)
   - Includes a reducer that checks the status and always returns "Under construction" (the initial state should check to that string)

❤Thank you!

### CODE REVIEW

File: src/redux/store.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { configureStore } from '@reduxjs/toolkit';
+import booksReducer from './books/booksSlice';
+
+const store = configureStore({
+  reducer: {
+    books: booksReducer,
+  },
```
Comment (Olamarx) [inline src/redux/store.js]: - [x] You must have forgotten to import categoriesSlice as you did for booksReducer.

File: src/redux/catagories/catagoriesSlice.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState: 'Under construction',
+  reducers: {},
```
Comment (Olamarx) [inline src/redux/catagories/catagoriesSlice.js]: - [x] Kindly setup the reducer too as it is required.

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @lilskyex0x, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Add and remove functions ✔️
- Readme file ✔️
- Linters passed ✔️
- Commit messages meaningful ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mikethreels — APPROVED** [review]
Hi Lilsnow, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- good job following the instructions of the previous CR.
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**